app/controllers/api/v2/preupgrade_reports_controller.rb

```diff
@@ -3,17 +3,17 @@
 module Api
   module V2
     class PreupgradeReportsController < ::Api::V2::BaseController
-      before_action :find_resource, only: %i[show]
```

Comment:
Why?

Reply:
Because now `@preupgrade_report` is searched by `find_by(job_invocation_id: params[:id])`, not by `find(params[:id])`.

Reply:
Right, thanks